### ISSUE TITLE
9.5 Staging 2191 - Store Left click then right click quickly audio two clicks but only one item added 

### ISSUE THREAD

**Issue by MishkaFur** [StrangeLoopGames/EcoIssues#22903]
Left click then right click in quick succession (or right click then left click) you get two audible clicks/prompts as if it is adding two to the basket but only one is actually added.

**Comment by tomesc99** [comment]
Hey @MishkaFur ,

Thank you very much for your report. Can you reproduce this issue again? I have been putting it to the test and I found out that both right click and left click are responding properly when using them individually, even when it is done very quickly. However, I discovered that if you click too fast and accidentally press both of them at the same time, the sound is reproduced but only one item is added to the cart.

**Comment by MishkaFur** [comment]
> Hey @MishkaFur ,
> 
> Thank you very much for your report. Can you reproduce this issue again? I have been putting it to the test and I found out that both right click and left click are responding properly when using them individually, even when it is done very quickly. However, I discovered that if you click too fast and accidentally press both of them at the same time, the sound is reproduced but only one item is added to the cart.

Thats correct, if you click left then right click very quickly you get two sounds.
